src/test/java/com/wikia/webdriver/elements/fandom/components/SearchInput.java

```diff
@@ -0,0 +1,41 @@
+package com.wikia.webdriver.elements.fandom.components;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import org.apache.xpath.operations.String;
+import org.openqa.selenium.By;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.By'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)